types/convict/index.d.ts

```diff
@@ -199,7 +200,7 @@ interface convict {
     addFormat(format: convict.Format): void;
     addFormats(formats: { [name: string]: convict.Format }): void;
     addParser(parsers: convict.Parser | convict.Parser[]): void;
-    <T>(config: convict.Schema<T> | string): convict.Config<T>;
+    <T>(config: convict.Schema<T> | string, options?: { env?: any, args?: string[] }): convict.Config<T>;
```

Comment:
I would prefer defining separate type and use the same typedef for `env` that `process.env` has, ie.

```
/// <reference types="node" />

...

    type Options = {
        env?: NodeJS.ProcessEnv;
        args?: string[];
    }

...

    <T>(config: convict.Schema<T> | string, opts?: convict.Options): convict.Config<T>;
```

Reply:
@vesse Thanks for the quick review! I agree, but switched out the type to an interface, I hope that's OK. I prefer to use interfaces when not specifically forced to use a type. Can't remember for sure but I may have read it in the official Typescript docs that this is a generally recommended approach.